SparkPost/2web2ui
# EO-628 &  EO-632 Health Score Dashboard

### What Changed
- EO-628 (#892) Current Health Gauge
- EO-632 (This PR) Health Score Timeseries & style changes
- https://sparkpost.invisionapp.com/share/H8R0LV8E5NA#/352294734_Health_Dashboard

### How To Test
- [Point your local upstreams to staging](https://confluence.int.messagesystems.com/display/ENG/Configuring+Openresty+Upstreams)
- Log into appteam or enable the `feature_signals_v2` UI option
- Navigate to the new Health Score page from the main side nav
- Verify the bar chart interaction (clicks, hovers) on other pages

### Todo
- [x] Merge in EO-632
